build-config/make/grub.make

```diff
@@ -211,23 +227,32 @@ GRUB_HOST_BIN_I386_DIR		= $(GRUB_HOST_INSTALL_I386_DIR)/usr/bin
 GRUB_HOST_INSTALL_UEFI_DIR	= $(GRUB_HOST_BUILD_DIR)/x86_64-efi-install
 GRUB_HOST_LIB_UEFI_DIR		= $(GRUB_HOST_INSTALL_UEFI_DIR)/usr/lib/grub/x86_64-efi
 GRUB_HOST_BIN_UEFI_DIR		= $(GRUB_HOST_INSTALL_UEFI_DIR)/usr/bin
-GRUB_HOST_CONFIGURE_I386_STAMP	= $(STAMPDIR)/grub-host-configure-i386-pc
-GRUB_HOST_CONFIGURE_UEFI_STAMP	= $(STAMPDIR)/grub-host-configure-x86_64-efi
 GRUB_HOST_CONFIGURE_STAMP	= $(STAMPDIR)/grub-host-configure
-GRUB_HOST_BUILD_I386_STAMP	= $(STAMPDIR)/grub-host-build-i386-pc
-GRUB_HOST_BUILD_UEFI_STAMP	= $(STAMPDIR)/grub-host-build-x86_64-efi
 GRUB_HOST_BUILD_STAMP		= $(STAMPDIR)/grub-host-build
 GRUB_HOST_INSTALL_STAMP		= $(STAMPDIR)/grub-host-install
+GRUB_HOST_CONFIGURE_I386_STAMP	= $(STAMPDIR)/grub-host-configure-i386-pc
+GRUB_HOST_BUILD_I386_STAMP	= $(STAMPDIR)/grub-host-build-i386-pc
+GRUB_HOST_INSTALL_I386_STAMP	= $(STAMPDIR)/grub-host-install-i386-pc
+GRUB_HOST_UEFI_ENABLE		= $(shell if [ "$(UEFI_ENABLE)" = "yes" ] || \
+				  [ "$(PXE_EFI64_ENABLE)" = "yes" ] ; then \
+				  echo "yes" ; else echo "no" ; fi)
```

Comment:
Not fan of shelling out to test variables.  How about using Makefile conditionals:

```
GRUB_HOST_UEFI_ENABLE = no
ifeq ($(UEFI_ENABLE),yes)
  GRUB_HOST_UEFI_ENABLE = yes
endif
ifeq ($(PXE_EFI64_ENABLE),yes)
  GRUB_HOST_UEFI_ENABLE = yes
endif
```